src/Storage/Repository/PgDataRepository.cs

```diff
@@ -0,0 +1,336 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Configuration;
+using Altinn.Platform.Storage.Interface.Models;
+using Azure;
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Npgsql;
+using NpgsqlTypes;
+
+namespace Altinn.Platform.Storage.Repository
+{
+    /// <summary>
+    /// Represents an implementation of <see cref="IDataRepository"/>.
+    /// </summary>
+    public class PgDataRepository : IDataRepository
+    {
+        private readonly string _insertSql = "call storage.insertDataelement ($1, $2, $3, $4)"; //"insert into storage.dataelements(instanceInternalId, instanceGuid, alternateId, element) VALUES ($1, $2, $3, $4);";
```

Comment:
```suggestion
         //"insert into storage.dataelements(instanceInternalId, instanceGuid, alternateId, element) VALUES ($1, $2, $3, $4);";
        private readonly string _insertSql = "call storage.insertDataelement ($1, $2, $3, $4)"; 
```
Move the comment to above the statement. use four / instead of two. See if that removes all these warnings. Repeat for the other statmements as well. You might be forced to add a blank line between statements.

Reply:
Done